sdk/java-sdk/src/main/scala/kalix/javasdk/impl/workflowentity/WorkflowEntityImpl.scala

```diff
@@ -258,8 +262,9 @@ final class WorkflowEntityImpl(system: ActorSystem, val services: Map[String, Wo
               case NonFatal(ex)       =>
                 // FIXME: not want we need.
                 // We need an exception with more context about the failed step
-                log.error(s"step execution protocol exception for $executeStep", ex)
-                throw ProtocolException(executeStep.stepName)
+                throw EntityException(
```

Comment:
we can now remove the FIXME here above and below

Reply:
done